pyvista/plotting/renderer.py

```diff
@@ -96,6 +97,133 @@ def viewup(self, value):
         self._viewup = value
 
 
+class Camera:
```

Comment:
We really should be subclassing here.

```py
class Camera(vtk.vtkCamera):
```

This follows the same approach we've used for the other classes (i.e. ``pyvista.Renderer`` so we can still pass objects of this class to vtk, and anyone wanting to use this class in a legacy manner still can.

Reply:
I fixed it. Thanks for your comment :+1: